monitoring/values.yaml

```diff
@@ -0,0 +1,246 @@
+global:
+  ingress:
+    # optional annotation overrides
+    annotations: {}
+
+grafanaIngress:
+  annotations:
+    alb.ingress.kubernetes.io/healthcheck-path: /api/health
```

Comment:
Having the `alb.ingres...` annotation here hard codes this new monitoring chart to the AWS ingress schema.  I don't know what the solution for this is, so i'm not going to suggest blocking the PR on solving this; just food for thought as this is going to be an issue when we try to deploy on Azure

Reply:
This has been changed to

```
# A place to define a healthcheck path or other grafana specific annotations
grafanaIngress:
  annotations: {}

# A place to define a healthcheck path or other prometheus specific
# annotations
prometheusIngress:
  annotations: {}
```

With healthchecks defined in the deploy repo. There should no AWS stuff in the public chart, so I think this fixes that but makes it clear that it may need to be filled in.